dev/VSIX/Extension/source.extension.vsixmanifest

```diff
@@ -14,17 +14,31 @@
     <Preview>false</Preview>
   </Metadata>
   <Installation AllUsers="true">
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.9, 17.0)" />
-    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[16.9, 17.0)" />
-    <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[16.9, 17.0)" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[16.9, 17.0)">
+      <ProductArchitecture>x86</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[17.0, 18.0)">
+      <ProductArchitecture>amd64</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[16.9, 17.0)">
+      <ProductArchitecture>x86</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[17.0, 18.0)">
+      <ProductArchitecture>amd64</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[16.9, 17.0)">
+      <ProductArchitecture>x86</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[17.0, 18.0)">
+      <ProductArchitecture>amd64</ProductArchitecture>
+    </InstallationTarget>
   </Installation>
   <Dependencies>
     <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
   </Dependencies>
   <Prerequisites>
-    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[16.0,17.0)" DisplayName="Visual Studio core editor" />
```

Comment:
Did this get renamed? Follow up and check on why this is needed, and if there's something this should be replaced with.

Reply:
I had a weird install of VS 2022 Preview that wasn't resolving this Prerequisite. Did some reinstalling/updating and things work again, so I was able to restore this prerequisite.